openapi/crucible-agent.json

```diff
@@ -439,6 +439,7 @@
           },
           "source": {
             "nullable": true,
+            "default": null,
```

Comment:
I'm guessing this is `schemars` unbreaking itself after that optional argument bug was introduced, but just checking.